README.md

```diff
@@ -1,3 +1,18 @@
+Supported by 
+----------
+<div align="center" markdown="1">
+
+[![Warp](https://github.com/user-attachments/assets/0c3b132c-8984-49cf-918a-db6f6abe2c01)](https://www.warp.dev/?utm_source=github&utm_medium=referral&utm_campaign=fzf)<br>
+
+### [Fzf's Terminal of Choice for Developers](https://www.warp.dev/?utm_source=github&utm_medium=referral&utm_campaign=fzf)
```

Comment:
Thanks. While I greatly appreciate Warp's support to my project, this sentence might feel confusing to the readers because `fzf` is listed as one of the incompatible tools in your documentation.

* https://docs.warp.dev/help/known-issues#list-of-incompatible-tools
* https://github.com/warpdotdev/Warp/issues/1647

In a previous email to Tess, I noted that "incompatible" is a strong word, because fzf itself works fine on Warp. What doesn't work is [the shell integration part](https://github.com/junegunn/fzf?tab=readme-ov-file#setting-up-shell-integration), which is technically an opt-in feature, albeit most users of fzf do enable it.

So until this issue is fixed, I think we should either omit this line or at least tone it down a little bit.